servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/RoundRobinSelector.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ * Copyright © 2023, 2024 Apple Inc. and the ServiceTalk project authors
```

Comment:
We have `scripts/update-copyright.sh` that ensures the format according to guidelines. 
For subsequent years, `-` should be used: `2023-2024`. 
If there is a gap, comma should be used: `2019, 2024`.